content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -7,75 +7,82 @@ reviewers:
 - liggitt
 content_type: concept
```

Comment:
Shall we separate server-side-apply changes into another PR?
We are mixing a lot of changes in this one.

Reply:
This is now mostly about SSA and patching.